### ISSUE TITLE
Submission to the trailer contest

### ISSUE THREAD

**Issue by gigapouch** [RigsOfRods/rigs-of-rods#377]
For this contest: http://www.rigsofrods.com/threads/119952-Trailer-contest-RoR-Promo-Contest-Trailer-0-4-5-0

I'm posting this here because it says that I'm not old enough for the RoR forums (seriously...) or Youtube:
https://vimeo.com/135985222

**Comment by Hiradur** [comment]
I noticed two things: the transition in the middle isn't really necessary and should be avoided. Though the background is very scenery is nice the vast plane of grass makes it look a bit bad.

**Comment by DirtGamer301** [comment]
The website doesn't load for me.

**Comment by Hiradur** [comment]
@DirtGamer301 Try right click->save as

**Comment by gigapouch** [comment]
I'll try a test terrain with pavement and trees/hills in the background, then. :)

**Comment by gigapouch** [comment]
@DirtGamer301 Lol, my "free" DNS service didn't update my IP address.

**Comment by Max98** [comment]
@gigapouch You sadly crossed the dead line. (10 August)

**Comment by gigapouch** [comment]
I read on the forums that there was no deadline! I was about to upload my new clip too.

**Comment by gigapouch** [comment]
How is this monster truck clip: https://vimeo.com/135995654

**Comment by DirtGamer301** [comment]
So...
a) You used an illegal vehicle (AMC Pacer) so it couldn't be an official trailer
b) The camera of the driving monster truck is boring
c) The Audi is bugging, so it also looks bad

**Comment by gigapouch** [comment]
How is that an illegal vehicle? I play with it all the time! I suppose I have other illegal vehicles as well.

**Comment by Hotrod55** [comment]
Any vehicles using meshes ripped from another game/person without permission is considered illegal. If it's not allowed on the RoR forums, it shouldn't be allowed on GitHub either.

**Comment by gigapouch** [comment]
But how am I supposed to know if it is legal or not? And how did you know? :/

**Comment by Max98** [comment]
If it wasn't released in the forum, then it's 99.99% illegal.

**Comment by gigapouch** [comment]
I've downloaded a lot of vehicles online other than the RoR website. Does that mean I have to get rid of them? I've noticed all of these "illegal" vehicles are by 09Challenger.

**Comment by Max98** [comment]
You can make/use as much as illegal vehicles as you want, as soon as you don't talk about them on official websites. (Keep them private just for you)

**Comment by gigapouch** [comment]
OK. I guess you're right, they also use stolen sounds from Gabester. :-1:

**Comment by skybon** [comment]
Let me rephrase what Moncef said:

> Rigs of Rods team will not condone copyright infringement. Illegal content posted on official resources will get you banned there. Otherwise it is completely on your head. Do not expect us to accept such material under any circumstances.
